DesignSystem/src/containers/Atoms.jsx

```diff
@@ -3,6 +3,9 @@ import SyntaxHighlighter from 'react-syntax-highlighter';
 import { dark } from 'react-syntax-highlighter/dist/esm/styles/prism';
 import components from './components';
 
+// LogoLoading
+import LogoLoading from '../Components/Atoms/LogoLoading';
```

Comment:
The import doesn't work. It can't resolve the path

Reply:
It's working here... Doesn't show any error in my Pc.
![Captura](https://user-images.githubusercontent.com/54256796/87073183-984a5780-c1e2-11ea-83be-04d05c88494c.PNG)